lib/jblond/Diff/Renderer/Text/UnifiedCli.php

```diff
@@ -63,7 +63,7 @@ public function render(): string
      */
     private function colorizeString($string, $color = ''): string
     {
-        if (isset($this->options['cliColor']) && $this->options['cliColor'] == 'simple') {
+        if (isset($this->options['cliColor']) && $this->options['cliColor'] === true) {
```

Comment:
I think `if ($this->options['cliColor']) {` is sufficient.

The variable should be set already by MainRendererAbstract and acting on any truthy value is ok.

Reply:
I changed that.